package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "nativeTemplate",
+  "name": "native-template",
   "version": "0.0.1",
```

Comment:
0.1.0 as proposed on the android PR.

Reply:
Rebasing from master when android is merged